_posts/2024-05-23-pyos-call-for-editors.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title: "Volunteers needed: pyOpenSci Editorial Board"
+excerpt: "pyOpenSci is looking for volunteers from a wide array of scientific backgrounds to join our Editorial Board."
+author: "Jesse Mostipak"
+permalink: /blog/pyos-call-for-editors-may-2024.html
+header:
+  overlay_image: images/blog/headers/23-05-2024-call-for-editors.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## Join the pyOpenSci Editorial Board
+Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteers to our Editorial Board.
+
+The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:
```

Comment:
```suggestion
The pyOpenSci Editorial Board is comprised of a diverse group of volunteers. As an editor, you join a diverse group of volunteers who care deeply about supporting the scientific Python software that scientists need to do their science.  You will lead 2-4 scientific Python package reviews a year.  The typical amount of time per review is anywhere from 1-3 hours a week. You may spend a bit more time up front finding reviewers (this is normally the trickiest part).

  As an editor, you are responsible for the following tasks:
```

Reply:
feel free to edit - i'm just trying to be more specific and answering the questions we always get. An editor also normally focuses on specific science domains.